.github/workflows/yosys-plugin.yml

```diff
@@ -0,0 +1,79 @@
+name: 'yosys-systemverilog-plugin'
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+
+  test-plugin:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: true
+    env:
+      CC: gcc-9
+      CXX: g++-9
+      DEBIAN_FRONTEND: noninteractive
+
+    steps:
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update -qq
+        sudo apt install -y software-properties-common
+        sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+        sudo apt-get update -qq
+        sudo apt install -y gcc-9 g++-9 build-essential cmake tclsh ant default-jre swig google-perftools libgoogle-perftools-dev python3 python3-dev python3-pip uuid uuid-dev tcl-dev flex libfl-dev git pkg-config libreadline-dev bison libffi-dev wget
```

Comment:
Surelog needs `python3-orderedmultidict` and `python3-psutil`, which are not mentioned here. Are they pulled in with `pip` ?

Reply:
`orderedmultidict` is installed through `pip` a few lines further. It seems that `psutil` is only needed for Surelog's regression tests and we're not running them here.